internal/server/server.go

```diff
@@ -134,7 +134,13 @@ func newGrpcServer(r registerGrpcHandlers, interceptors ...grpc.UnaryServerInter
 		// Add tags to logger context.
 		grpcinterceptor.UnaryServerLoggerInterceptor(),
 		// Todo: Confirm tracing is working as expected. Jaegar integration?
-		grpcopentracing.UnaryServerInterceptor(),
+		grpcopentracing.UnaryServerInterceptor(grpcopentracing.WithFilterFunc(func(ctx context.Context, fullMethodName string) bool {
+			if fullMethodName == "/google.pubsub.v1.StatusCheckAPI/LivenessCheck" ||
```

Comment:
better to maintain a running list of URLs as a constant somewhere. This list may/can expand in the future.